dashboard/android/key.properties

```diff
@@ -0,0 +1,4 @@
+storePassword=dartdart
+keyPassword=dartdart
+keyAlias=upload
+storeFile=$HOME/upload-keystore.jks
```

Comment:
Make sure you store this file somewhere secure that's able to be ACL'd by other Googlers (like a password storage system)

Reply:
Done! I updated the README with instructions for the one time set up.